src/ClassAliasAutoloader.php

```diff
@@ -45,21 +66,14 @@ public static function register(Shell $shell, $classMapPath)
     public function __construct(Shell $shell, $classMapPath)
     {
         $this->shell = $shell;
-
-        $vendorPath = dirname(dirname($classMapPath));
+        $this->vendorPath = dirname(dirname($classMapPath));
+        $this->includedAliases = collect($GLOBALS['config']['tinker.alias'] ?? []);
```

Comment:
We need to change this to be injected into the constructor.

Reply:
Why did you change this? This was fine.

Reply:
Neither work, because the tests can't pass without this change. The problem is the use of the config helper at all.

Reply:
The tests were passing.

Reply:
The tests are passing but the code still doesn't actually work.

Reply:
Maybe I will take over this PR later tonight on my machine. The changes I was able to make from the github.com UI don't fully address the problem.

Reply:
What exactly was the problem in the first place? It seemed to work fine.

Reply:
The problem is the config function doesn;t exist in general.

Reply:
It does in Laravel.

Reply:
This package doesn't depend on Laravel. It only depends on https://github.com/laravel/tinker/blob/2.x/composer.json#L14-L16.

Reply:
How do you use this package without Laravel?

Regardless, this PR isn't the thing that's introducing the use of the `config()` helper. It was already being used.